config/src/main/java/org/axonframework/config/EventProcessingConfigurer.java

```diff
@@ -260,6 +260,8 @@ EventProcessingConfigurer registerTokenStore(String processorName,
      */
     EventProcessingConfigurer usingPooledStreamingEventProcessors();
 
+    EventProcessingConfigurer usingEventProcessorBuilder(EventProcessorBuilder processorBuilder);
```

Comment:
Missing some JavaDoc here too! I might help you out here though.

Reply:
what would you recommend to write here?